Makefile

```diff
@@ -44,6 +44,10 @@ fmt:
 vet:
 	go vet ./pkg/... ./cmd/...
 
+# Run go fmt Test
+gofmt:
```

Comment:
I like the idea of moving the test into Makefile. However, the `gofmt` name above is too similar to l.40, making it hard to understand what's the difference. Seeing this, my proposal is as follows:

1. remove this `gofmt` target, moving its contents (the test line 49) into the `vet` target above
2. move the `golint` call from .travis.yml into the `vet` target above
3. rename the `vet` target to `lint`
    - including all usages of `vet` in the repository

Reply:
the `vet` is calling at multiple places in Makefile. In that case, we have to update that as well. But the lint will be called multiple times at multiple places.

Reply:
Hmm; you're right, this is not so clear cut, for example for quick testing during development we may not want to call full lint (with its complaints about missing godoc comments etc.).

Ok, so I'll back off to just ask you to rename `gofmt` target to `check-gofmt`.

Thanks!

Reply:
Done